packages/flutter/test/material/scaffold_test.dart

```diff
@@ -998,6 +998,56 @@ void main() {
       semantics.dispose();
     });
 
+    testWidgets('Drawer state query correctly', (WidgetTester tester) async {
```

Comment:
Test should also verify that the new properties work correctly when the drawers are opened with openDrawer(), openEndDrawer()